examples/di/InfiAgent-DABench/run_InfiAgent-DABench_sigle.py

```diff
@@ -0,0 +1,22 @@
+import fire
+from DABench import DABench
+
+from metagpt.logs import logger
+from metagpt.roles.di.data_interpreter import DataInterpreter
+from metagpt.utils.recovery_util import save_history
+
+
+async def main(id=0):
+    """Evaluate one task"""
+    DA = DABench()
```

Comment:
I prefer to use **bench** as a variable. Generally speaking, python PEP8 does not allow uppercase variable names

Reply:
done